enforcer-rules/src/main/java/com/google/cloud/tools/opensource/enforcer/LinkageCheckerRule.java

```diff
@@ -146,8 +149,12 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
           new DefaultDependencyResolutionRequest(mavenProject, session);
       DependencyResolutionResult dependencyResolutionResult =
           projectDependenciesResolver.resolve(dependencyResolutionRequest);
-      return dependencyResolutionResult.getDependencies().stream()
```

Comment:
`dependencyResolutionResult.getDependencies()` does not return Maven's breadth-first traversal order.